src/Server/HTTPHandlerFactory.cpp

```diff
@@ -73,11 +86,40 @@ static auto createPingHandlerFactory(IServer & server)
     {
         constexpr auto ping_response_expression = "Ok.\n";
         return std::make_unique<StaticRequestHandler>(
-            server, ping_response_expression, parseHTTPResponseHeaders("text/html; charset=UTF-8"));
+            server, ping_response_expression, parseHTTPResponseHeaders("text/plain; charset=UTF-8"));
```

Comment:
your variant is formally more correct / better. But let's avoid touching it to avoid some non-planned side-effects (can be used in healthchecks).